### PR TITLE
Additions to CHANGELOG

### PR DESCRIPTION
**Pull request by adam-fowler**
I have assume #57 will be merged

### CODE REVIEW

**Review by 0xTim — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,10 +4,19 @@ All notable changes to this project will be documented in this file.
...
 
 ### Added
 
-- Automatically create tasks based on the targets in a package.
+- Automatically create build tasks based on the targets in a package.
 - Show package dependencies in the Explorer.
+- Add button to package dependencies view to update packages.
 - Resolve dependencies when **Package.swift** or **Package.resolved** changes.
+- Integrated with Soucekit-LSP.
+- Generate launch configurations for each executable and tests in Package.swift.
```
Comment (svanimpe) [inline CHANGELOG.md:16]:
```suggestion
- Generate launch configurations for each executable and tests in **Package.swift**.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,10 +4,19 @@ All notable changes to this project will be documented in this file.
...
+- Added "Swift" output channel providing a history of all actions.
+- Add status bar item for when swift resolve or updates are running.
```
Comment (svanimpe) [inline CHANGELOG.md:18]:
```suggestion
- Add status bar item for when resolve or update tasks are running.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,10 +4,19 @@ All notable changes to this project will be documented in this file.
...
+## 0.1.0
 
 ### Added
 
-- Automatically create tasks based on the targets in a package.
+- Automatically create build tasks based on the targets in a package.
 - Show package dependencies in the Explorer.
+- Add button to package dependencies view to update packages.
 - Resolve dependencies when **Package.swift** or **Package.resolved** changes.
+- Integrated with Soucekit-LSP.
```
Comment (svanimpe) [inline CHANGELOG.md:15]:
```suggestion
- Integrate the SourceKit-LSP extension.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,10 +4,19 @@ All notable changes to this project will be documented in this file.
...
+- Generate launch configurations for each executable and tests in Package.swift.
+- Added "Swift" output channel providing a history of all actions.
```
Comment (svanimpe) [inline CHANGELOG.md:17]:
```suggestion
- Add "Swift" output channel providing a history of all actions.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,10 +4,19 @@ All notable changes to this project will be documented in this file.
...
+- Add status bar item for when swift resolve or updates are running.
+- Bundle using ESBuild.
+- Add configuration parameter `path` to define where swift is found.
```
Comment (svanimpe) [inline CHANGELOG.md:20]:
```suggestion
- Add configuration parameter `path` to define where `swift` is found.
```

**Review by svanimpe — APPROVED** [review]
Just some small formatting / consistency nits, otherwise good to go!

File: CHANGELOG.md (hunk 3 above)
Comment (adam-fowler) [reply CHANGELOG.md:15]: That implies it is a separate extension.

File: CHANGELOG.md (hunk 4 above)
Comment (adam-fowler) [reply CHANGELOG.md:17]: Thinking about it. Shouldn't all the lines be in the past tense. Given the section is called "Added".

File: CHANGELOG.md (hunk 5 above)
Comment (adam-fowler) [reply CHANGELOG.md:20]: Not sure we need to add marks around swift but it should be capitalised.

**Pull request merged**